starters/basic-starter/pages/index.tsx

```diff
@@ -88,6 +89,23 @@ export default function IndexPage({ menus, events, places }: IndexPageProps) {
 export async function getStaticProps(
   context,
 ): Promise<GetStaticPropsResult<IndexPageProps>> {
+  let index;
+  try {
+    index = await drupal.getIndex();
+  } catch (e) {
+    throw new Error(
+      e +
+        '\nSee here for documentation on how to debug common failures: https://github.com/acquia/next-acms/wiki/Debugging-common-errors-with-Next.js-and-Drupal-integration#incompatible-web-server-setup-or-incorrect-environment-variables-generated',
```

Comment:
I think we could benefit from a second new line here. It would separate this more and make it easier to read.

Reply:
I think we should also explicitly say that this error is about failing to connect to the backend since that by itself is pretty helpful.